src/NuGet.Clients/VisualStudio/Extensibility/IVsPackageInstallerServices.cs

```diff
@@ -37,7 +38,7 @@ public interface IVsPackageInstallerServices
         /// <param name="id">The id of the package to check.</param>
         /// <param name="version">The version of the package to check.</param>
         /// <returns><c>true</c> if the package is install. <c>false</c> otherwise.</returns>
-        bool IsPackageInstalled(Project project, string id, SemanticVersion version);
+        bool IsPackageInstalled(Project project, string id, NuGetVersion version);
```

Comment:
This can't change, it has to stay as a SemanticVersion. The NuGetVersion method should just be internal on the class and not part of this interface.